blueflood-http/src/main/java/com/rackspacecloud/blueflood/outputs/serializers/BasicRollupsOutputSerializer.java

```diff
@@ -101,8 +101,8 @@ Object convertRawSampleToObject(Object rawSample) {
         NUM_POINTS("numPoints") {
             @Override
             Object convertRollupToObject(Rollup rollup) throws Exception {
-                if (rollup instanceof BasicRollup)
-                    return ((BasicRollup) rollup).getCount();
+                if (rollup instanceof BaseRollup)
+                    return ((BaseRollup) rollup).getCount();
```

Comment:
BluefloodGaugeRollup always sub-classed BasicRollup, so it would have been caught in the pre-existing  if ( rollup instanceof BasicRollup) check, right?  If so, then numPoints already existed for Gauge.